manifests/init.pp

```diff
@@ -107,7 +107,7 @@
   Stdlib::Absolutepath            $configfile           = '/etc/opendkim.conf',
   Stdlib::Absolutepath            $pidfile              = '/run/opendkim/opendkim.pid',
   Pattern[/\A[0-7]{3,4}\z/]       $rundir_mode          = '0755',
-  Optional[Stdlib::Absolutepath]  $sysconfigfile        = '/etc/sysconfig/opendkim', # lint:ignore:optional_default
+  Optional[Stdlib::Absolutepath]  $sysconfigfile        = undef,
```

Comment:
The actual case where this is useful is provided by Hiera:

https://github.com/lvicainne/puppet-opendkim/blob/e2ce5dd112e7b8470e85c5a67f2c34deacbdba67/data/RedHat.yaml#L2